cpu/nrf52/Makefile.include

```diff
@@ -1,5 +1,9 @@
 export CPU_ARCH = cortex-m4f
 export CPU_FAM  = nrf52
 
+# Export internal ROM alignment and image header size for bootloader support
+export IMAGE_ALIGN = 8
+export IMAGE_HDR_SIZE = 512
```

Comment:
I understand, that the `IMAGE_HDR_SIZE` can have different values, but these don't have anything to do with the board, but are dependent on CPU configuration. And as RIOT does not support putting the VTOR into RAM (yet), we don't need to worry about it. So I think this variable should be defined with a default value somewhere in the `cpu/cortexm_common` path instead of in the board path...